toolbox/githubctl/main.go

```diff
@@ -220,6 +221,7 @@ func ReleasePipelineQualification(baseBranch *string) error {
 	u.AssertNotEmpty("hub", hub) // TODO (chx) default value of hub
```

Comment:
there are way too much release specific information in this util.

perhaps this should be updated to create a new PR with some basic input?